test/integration/runner_task_test.rb

```diff
@@ -7,18 +7,83 @@ def test_run_works
     deploy_fixtures("hello-cloud", subset: ["template-runner.yml"])
 
     task_runner = build_task_runner
-    assert task_runner.run(**valid_run_params)
+    refute task_runner.run(**valid_run_params)
```

Comment:
why this change?

Reply:
Because the `run`method didn't actually verify success, it turns these tests didn't. I've fixed the tests so that they do actually run successfully